File: banchus-common/src/main/java/pe/nanamochi/banchus/domain/dto/LoginData.java
package pe.nanamochi.banchus.domain.dto;

public record LoginData(
    String username,
    String passwordMd5,
    String osuVersion,
    int utcOffset,
    boolean displayCity,
    boolean pmPrivate,
    String osuPathMd5,
    String adaptersStr,
    String adaptersMd5,
    String uninstallMd5,
    String diskSignatureMd5) {}

File: banchus-app/src/main/kotlin/pe/nanamochi/banchus/dto/LoginData.kt
package pe.nanamochi.banchus.dto

data class LoginData(
    val username: String,
    val passwordMd5: String,
    val osuVersion: String,
    val utcOffset: Int,
    val displayCity: Boolean,
    val pmPrivate: Boolean,
    val osuPathMd5: String,
    val adaptersStr: String,
    val adaptersMd5: String,
    val uninstallMd5: String,
    val diskSignatureMd5: String,
)
